src/components/ChatEntry.js

```diff
@@ -1,22 +1,42 @@
 import React from 'react';
+import { useState } from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+
+  const [color, setColor] = useState(false);
```

Comment:
Our `chatMessages` data already has a `liked` attribute for each chat entry, so we should be utilizing that instead of making a new piece of state that is separate from our single source of truth. 

Now we have contradicting data that can confuse teammates and make it more difficult to catch bugs.